--- a/data/meson.build
+++ b/data/meson.build
@@ -23,7 +23,7 @@ appstream_file = i18n.merge_file(
   output: app_id + '.appdata.xml',
   po_dir: '../po',
   install: true,
-  install_dir: join_paths(get_option('datadir'), 'appdata')
+  install_dir: join_paths(get_option('datadir'), 'metainfo')
 )
 
 icondir = join_paths(get_option('datadir'), 'icons/hicolor/scalable/apps')
--- a/data/org.berarma.Oversteer.desktop.in
+++ b/data/org.berarma.Oversteer.desktop.in
@@ -6,5 +6,5 @@ Terminal=false
 Type=Application
 StartupNotify=true
 Icon=org.berarma.Oversteer
-Categories=Games;Utility;
+Categories=Game;Utility;
 Keywords=game;racing;cars;wheels
